wit/bindgen/generator.go

```diff
@@ -826,6 +826,20 @@ func (g *generator) enumRep(file *gen.File, dir wit.Direction, e *wit.Enum, goNa
 		b.WriteRune('\n')
 	}
 	b.WriteString(")\n")
+	b.WriteString("\n")
+
+	sliceName := "strings" + goName
```

Comment:
Should declare this name with file.DeclareName.

I suggest calling it stringsName, as it's not technically a slice.